packages/@uppy/aws-s3-multipart/src/index.js

```diff
@@ -32,22 +166,32 @@ export default class AwsS3Multipart extends BasePlugin {
     this.#client = new RequestClient(uppy, opts)
 
     const defaultOptions = {
-      timeout: 30 * 1000,
       limit: 0,
```

Comment:
our default limit is `0`? Should probably be something else

Reply:
For another PR though